README.md

```diff
@@ -10,7 +12,20 @@
 [![AUR version](https://img.shields.io/aur/version/tfsec-bin)](https://aur.archlinux.org/packages/tfsec-bin)
 [![VScode Extension](https://img.shields.io/visual-studio-marketplace/v/tfsec.tfsec?label=vscode)](https://marketplace.visualstudio.com/items?itemName=tfsec.tfsec)
 
-tfsec uses static analysis of your terraform code to spot potential misconfigurations.
+    
+> 📣 All of our scanning-related efforts have moved to one place, and that is Trivy. Please look at the [migration guide](https://aquasecurity.github.io/trivy/latest/tutorials/tfsec/migration/) and the Trivy [docs](https://aquasecurity.github.io/trivy/latest/) for further information.
```

Comment:
This migration link is 404ing, should it be: https://github.com/aquasecurity/tfsec/blob/master/tfsec-to-trivy-migration-guide.md